src/table.css

```diff
@@ -0,0 +1,201 @@
+ td, th {  
+  border: 1px solid #74756E;
+  border-left: 0px;
+  border-top: 0px;
+  text-align: left;
+}
+
+table {
```

Comment:
Ditto these tag styles - we should add a class

Reply:
I'll add a class to the table and use that to make the descendant CSS rules more specific. Thank you for catching these!